packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewEvmExplanation.tsx

```diff
@@ -29,16 +18,14 @@ export const TransactionReviewEvmExplanation = ({
     }
 
     return (
-        <Wrapper $margin={margin}>
```

Comment:
Good example how not use `frameProps`. It should available be only for internal usage in DS

Reply:
Can you please elaborate? I'm still not sure what's the issue here.

Reply:
It has zero additional value and it's easily replaceable with frame props in `Warning` component. Also I don't like that it's imported outside of `@trezor/component` API (`@trezor/components/src/components/common/frameProps`).

Reply:
Gotcha, thanks!